autovizwidget/autovizwidget/__init__.py

```diff
@@ -1 +1 @@
-__version__ = '0.12.4'
+__version__ = '0.12.5'
```

Comment:
You need to bump all libraries together, since we always push all to pip, and build would fail otherwise.

Reply:
Done!

Reply:
Actually, sorry to have missed this, but we should ideally bump the 12 to 13 altogether. So new version would be 0.13.0, according to rules of semantic versioning. Can you please bump to 0.13.0 instead?